src/components/Header/SpacesDropdown.tsx

```diff
@@ -29,15 +30,9 @@ export function SpacesDropdown(props: { space: Space; spaces: Space[] }) {
             )}
         >
             <DropdownMenu>
-                {spaces.map((otherSpace) => (
-                    <DropdownMenuItem
-                        key={otherSpace.id}
-                        href={otherSpace.urls.published ?? otherSpace.urls.app}
-                        active={otherSpace.id === space.id}
-                    >
-                        {otherSpace.title}
-                    </DropdownMenuItem>
-                ))}
+                {spaces.map((otherSpace, index) => 
+                    <SpacesDropdownMenuItem key={`${otherSpace.id}-${index}`} currentSpace={space} variantSpace={otherSpace} />
```

Comment:
```suggestion
                    <SpacesDropdownMenuItem key={otherSpace.id} currentSpace={space} variantSpace={otherSpace} />
```

the id should be unique I guess?

Reply:
I have set the key to `${otherSpace.id}-${index}` instead of just `otherSpace.id` because with sites there could be 2 site spaces that points to the same space so using the space id will not make it unique.